test/units/TestUtils.py

```diff
@@ -512,15 +512,15 @@ def test_make_su_cmd(self):
         self.assertTrue('echo SUDO-SUCCESS-' in cmd[0] and cmd[2].startswith('SUDO-SUCCESS-'))
 
     def test_to_unicode(self):
-        uni = ansible.utils.to_unicode(u'ansible')
+        uni = ansible.utils.unicode.to_unicode(u'ansible')
         self.assertTrue(isinstance(uni, unicode))
         self.assertEqual(uni, u'ansible')
 
-        none = ansible.utils.to_unicode(None)
+        none = ansible.utils.unicode.to_unicode(None)
         self.assertTrue(isinstance(none, type(None)))
```

Comment:
This one will fail with the new to_unicode.   With no arguments, this will return ```"None"```` (the equivalent of ```str(none)``` ).  If we want the test to return None, then update like this:

``` python
none = ansible.utils.unicode.to_unicode(None, nonstring=passthru)
```